README.md

```diff
@@ -1,31 +1,332 @@
-# OOP Foundations Lab
+# OOP Fundamentals Lab
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Create a class hierarchy using inheritance
+- Override methods in the subclass
+- Call superclass methods from the subclass
+- Create classes that implement an interface
 
-## Introduction
+## Instructions
 
-A 1-2 sentence summary of what will be covered.
+Fork and clone this lab.
 
-## Instructions
 
-Walk the student through any setup required to run the lesson (i.e.
-`npm install` and `npm start`).
+## Task #1 - Do not alter the `Athlete` class
+
+You will implement the following class hierarchy:
+
+- Every athlete has a jersey number.
+- Football players are athletes that make touchdowns.
+- Hockey players are athletes that score goals and get penalties.
+- An athlete's performance rating is calculated based on the type of athlete:
+    - The rating for a football player is calculated as (touchdowns * 5).
+    - The rating for a hockey player is calculated as (goals * 4 - penalties * 2).
```

Comment:
As a fan of sports, these ratings are rough! Lol! Football players also can get penalties too but I get it for the sake of the assignment haha.

Reply:
Made up some random calculations.  Let me know if there is something else that makes more sense.